powerauth-java-server/src/main/java/com/wultra/security/powerauth/app/server/controller/api/v4/AuthenticationController.java

```diff
@@ -131,5 +143,54 @@ public ObjectResponse<VerifyOfflineAuthenticationResponse> verifyOfflineAuthenti
         return response;
     }
 
+    @Deprecated
```

Comment:
nitpick: deprecated should be marked whether it is for removal and described in JavaDoc why it is deprecated and what to use instead.